### DISCUSSION TITLE
Add: show git commands in Git Demonstration after execution

### DISCUSSION THREAD

**Discussion by tomi-neuron** [pcottle/learnGitBranching#1149] [Ideas]
Amazing visual demonstration in the Git Description window, however would you consider keeping the just executed git commands visible alongside the reset button? that way one wouldn't have to go back and forth to remind themselves of the commands we just executed (often multiple commands with ; separation is not always easy to remember exactly for a newcomer). Thank you very much!

**Comment by pcottle** [comment]
Good point! I went ahead and cleaned up this interaction -- now the command will stick around and we'll just have a small link for the reset animation part.
<img width="1058" alt="Screenshot 2024-05-27 at 9 49 11 PM" src="https://github.com/pcottle/learnGitBranching/assets/1135007/61b987b1-7b54-4218-a18a-5dbf58a608e4">

I had to tighten up the spacing a bit here but I think this should be better UX in general :) let me know what you think!

**Reply by tomi-neuron** [reply]
genius, thank you! :)
